tsconfig-strict.json

```diff
@@ -136,8 +136,6 @@
     "core/templates/services/debouncer.service.spec.ts",
     "core/templates/services/editability.service.ts",
     "core/templates/services/editability.service.spec.ts",
-    "core/templates/services/exploration-features.service.ts",
-    "core/templates/services/exploration-features.service.spec.ts",
```

Comment:
Why do you need to remove this?

Reply:
30+

Reply:
yeah, with a little help. If I have to do that, I would like to do it in a separate PR. This PR is blocking other directives/services of the exploration player page on which I'm currently working. Removing these entries from tsconfig-strict.json doesn't undo anything as exploration-features service file is still strictly typed but two newly added dependency are not, which were neither earlier.

Reply:
Okay, please add an issue and create a PR after you merge this.

Reply:
done

Reply:
Next time please provide the issue number.

Reply:
I think, I referenced this PR in the issue, that should add a feed referencing the issue here. But still will take care next time.